src/app.js

```diff
@@ -0,0 +1,30 @@
+import "dotenv/config";
+import express from "express";
+
+import authRouter from "./routes/auth-router.js";
+import usersRouter from "./routes/users-router.js";
+import productsRouter from "./routes/products-router.js";
+import postsRouter from "./routes/posts-router.js";
```

Comment:
잘못된 import 경로인 것 같습니다.
```suggestion
import authRouter from "./router/auth-router.js";
import usersRouter from "./router/users-router.js";
import productsRouter from "./router/product-router.js";
import postsRouter from "./router/posts-router.js";
```